### ISSUE TITLE
iOS LoginPage example is confusing

### ISSUE THREAD

**Issue by mokagio** [calabash/x-platform-example#4]
Hello, thanks for putting this example project together.

I have a little feedback, the [`login` method](https://github.com/calabash/x-platform-example/blob/master/features/ios/pages/login_page.rb#L17) of the `LoginPage` for iOS is a bit confusing. 

I understand that using the current approach might be faster, but being _non standard_ it add mental overhead on top of the POP architecture, which is already non trivial to grasp at first glance, at least for me. 

It might be better to go for the default approach, so that new comers don't have to worry about the side effects and the parameters values.

**Comment by krukow** [comment]
@mokagio: do you mean the use of fast keyboard entry?

```
 enter_text(user_field, user, use_keyboard: false)
    enter_text(pass_field, pass, use_keyboard: false, wait: false)
    enter_text(site_field, site, use_keyboard: false, wait: false)

    # If we used use_keyboard: true, this wouldn't be necessary
    # but use_keyboard: false is faster so we use it here
    touch("UITextField") # side-effect to have Add Site button enable
                      # after fast keyboard entry (= UIAutomation setValue)
```

**Comment by mokagio** [comment]
Yes I'm referring to that



—
Gio

**Comment by krukow** [comment]
I agree. It is not great for learning. I don't have time right now to change it- would you be interested in making a pull request that changes it?

**Comment by mokagio** [comment]
Hey @krukow sorry for this late reply. I was hoping to find some time to look into this and submit a PR but I haven't yet, and I don't think I'll have time in the next month either :confused: 

Thanks, and keep up the great work on Calabash :beers:
